### DISCUSSION TITLE
Feature Request: Having the projects tab in the Repo

### DISCUSSION THREAD

**Discussion by GrnPero** [community/community#46] [Mobile]
I would love to continue adding goals ideas etc. To the projects tab in my GitHub repo though the app. 
![GitHub Androiud](https://user-images.githubusercontent.com/44035269/98044916-87020800-1ddc-11eb-8450-19190ca7ccdd.PNG)

**Comment by candyho** [comment]
Hey, thanks for sharing your feedback! Better Projects support is something already on our radar. Adding an Issue to existing projects is already supported on mobile, you just need to slide up the "comment sheet" on the Issue view, look for the "Projects" section and tap "Edit". 

Other than being able to add your goals to existing projects, what are your other expectations to use projects on mobile?
